src/routes/news/components/Tabs.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components'
+import { CityModel } from '@integreat-app/integreat-api-client'
+import type { TFunction } from 'react-i18next'
+import Tab from './Tab'
+
+const StyledTabs = styled.div`
+  display: flex;
+  padding-top: 45px;
+  padding-bottom: 40px;
+`
+
+const LOCAL_NEWS = 'local'
+const TU_NEWS = 'tu'
+
+type PropsType = {|
+  localNews: boolean,
+  tunews: boolean,
+  children: React.Node,
+  city: string,
+  cities: Array<CityModel>,
+  language: string,
+  t: TFunction
+|}
+
+class Tabs extends React.PureComponent<PropsType> {
```

Comment:
I think we don't need for Tabs component, since Tabs is a generic component also Tabs items usually should have same style, but in our case we have two special buttons with different styles, so i suggest to create two components, one for LocalNewsLink and another one for TuNewsLink.

What do you think?